test/acceptance/Dockerfile.allure

```diff
@@ -0,0 +1,5 @@
+FROM yhxue911/python-allure
+
+RUN mkdir -p /allure/results && mkdir -p /allure/report
```

Comment:
you do not need to create these folder since you are mounting them always from host in `generate-allure-report.sh`

Reply:
good point, thx.